<launch>
    <node pkg="blue_rov_custom_integration" type="pp_node" name="Path_Planner" launch-prefix="gnome-terminal -e" />
    <node pkg="blue_rov_custom_integration" type="Controller_node" name="Controller" />
</launch>